### ISSUE TITLE
'undefined' was appended on the comment line when align across comments

### ISSUE THREAD

**Issue by seoker** [adrianlee44/atom-aligner#23]
Hi, I've tried the 'Allow across comments' option, two problems:
(1) when the selection include a comment at the first line, the aligner doesn't work.
(2) the aligner appends some 'undefined' at the end of comment line.
ex:
```javascript
        // URL parameter missing
        UrlParameterMissing: 0x10000000,
        UserIdMissing:       0x10000001,
        CarIdMissing:     0x10000002,
        // Body parameter missing
        BodyParameterMissing: 0x11000000,
        MobileMissing:        0x11000001,
        PasswordMissing:   0x11000002,
        OtpMissing:         0x11000003,
        EmailMissing:     0x11000004,
        BirthdayMissing:      0x11000005,
        // Bad format
        BadMobileFormat:      0x12000001,
        BadOtpFormat:         0x12000002,
        BadEmailFormat:       0x12000003,
        BadBirthdayFormat:    0x12000004,
```
result:
```javascript
        // URL parameter missing
        UrlParameterMissing:  0x10000000,
        UserIdMissing:        0x10000001,
        CarIdMissing:         0x10000002,
        // Body parameter missingundefined 
        BodyParameterMissing: 0x11000000,
        MobileMissing:        0x11000001,
        PasswordMissing:      0x11000002,
        OtpMissing:           0x11000003,
        EmailMissing:         0x11000004,
        BirthdayMissing:      0x11000005,
        // Bad formatundefined        
        BadMobileFormat:      0x12000001,
        BadOtpFormat:         0x12000002,
        BadEmailFormat:       0x12000003,
        BadBirthdayFormat:    0x12000004,
```

**Comment by adrianlee44** [comment]
Thanks for reporting this. I'll look into this asap.

**Comment by adrianlee44** [comment]
Patched above bugs with `v0.15.0`. Please let me know if you run into any more issues and thanks for reporting them.
